modules/schematics/src/feature/index.ts

```diff
@@ -9,51 +9,69 @@ import { Schema as FeatureOptions } from './schema';
 
 export default function (options: FeatureOptions): Rule {
   return (host: Tree, context: SchematicContext) => {
-    return chain([
-      schematic('action', {
-        flat: options.flat,
-        group: options.group,
-        name: options.name,
-        path: options.path,
-        project: options.project,
-        skipTests: options.skipTests,
-        api: options.api,
-        prefix: options.prefix,
-      }),
-      schematic('reducer', {
-        flat: options.flat,
-        group: options.group,
-        module: options.module,
-        name: options.name,
-        path: options.path,
-        project: options.project,
-        skipTests: options.skipTests,
-        reducers: options.reducers,
-        feature: true,
-        api: options.api,
-        prefix: options.prefix,
-      }),
-      schematic('effect', {
-        flat: options.flat,
-        group: options.group,
-        module: options.module,
-        name: options.name,
-        path: options.path,
-        project: options.project,
-        skipTests: options.skipTests,
-        feature: true,
-        api: options.api,
-        prefix: options.prefix,
-      }),
-      schematic('selector', {
-        flat: options.flat,
-        group: options.group,
-        name: options.name,
-        path: options.path,
-        project: options.project,
-        skipTests: options.skipTests,
-        feature: true,
-      }),
-    ])(host, context);
+    return chain(
+      [
+        schematic('action', {
+          flat: options.flat,
+          group: options.group,
+          name: options.name,
+          path: options.path,
+          project: options.project,
+          skipTests: options.skipTests,
+          api: options.api,
+          prefix: options.prefix,
+        }),
+        schematic('reducer', {
```

Comment:
That makes sense! I may have been focused on the small stuff rather than the overall flow of the generator. I resorted to using `MergeStrategy.Overwrite` to try and avoid duplicated actions/reducers, which in retrospect, is a relatively aggressive way of handling code generation. 

I'll try this approach and update the PR.

Reply:
Instead of overwriting could we also skip the reducer and action schematics when using the `entity` flag?